langlib/lang.table/src/main/ballerina/table.bal

```diff
@@ -110,6 +168,14 @@ public isolated function add(table<MapType> t, MapType val) = @java:Method {
 
 # Applies a function to each member of a table and returns a table of the result.
 #
+# ```ballerina
+# table [
+#     {id: 1, maths: 78, physics: 70},
+#     {id: 2, maths: 83, physics: 80}
+# ]
+# .map((student) => {id: student.id, avg: (student.maths + student.physics) / 2}) ⇒ [{"id":1,"avg":74},{"id":2,"avg":81}]
```

Comment:
Also was it a consensus decision to bring the function call(`.map(...)`) to the next line?  @MaryamZi thoughts?

Reply:
+1 to have it on the same line as `]`. That's what we did for array too, but there everything was on one line.